splunk-otel-android/src/main/java/io/opentelemetry/rum/internal/ActiveSpan.java

```diff
@@ -14,30 +14,32 @@
  * limitations under the License.
  */
 
-package com.splunk.rum;
+package io.opentelemetry.rum.internal;
+
+import static io.opentelemetry.rum.internal.RumConstants.LAST_SCREEN_NAME_KEY;
 
 import androidx.annotation.Nullable;
 import io.opentelemetry.api.trace.Span;
 import io.opentelemetry.context.Scope;
 import java.util.function.Supplier;
 
-class ActiveSpan {
-    private final VisibleScreenTracker visibleScreenTracker;
+public class ActiveSpan {
```

Comment:
WDYT about moving this to `...util`?

Reply:
oh sure. The name is more data class than utility, but it does wrap and assist so that seems fine.